tests/foreman/ui/test_reporttemplates.py

```diff
@@ -393,6 +393,42 @@ def test_positive_schedule_generation_and_get_mail(session):
                       The result is compressed.
     :CaseImportance: High
     """
+    # make sure we have some subscriptions
+    with manifests.clone() as manifest:
+        upload_manifest(module_org.id, manifest.content)
+    # generate Subscriptions report
+    with session:
+        session.reporttemplate.schedule(
+                "Subscriptions", values={'output_format': 'JSON', 'generate_at': '1970-01-01 17:10:00', 'email': True, 'email_to': 'root@localhost'}
+        )
+    file_path = '/tmp/{0}.json'.format(gen_string('alpha'))
+    gzip_path = '{0}.gz'.format(file_path)
+    expect_script = '''#!/usr/bin/env expect
+spawn mail
```

Comment:
Could you use parentheses to create string spanning multiple lines, or make that a list that is then `"\n".join()`ed? So we don't have sudden de-indentation in middle of method.